pkg/cluster/internal/providers/docker/util.go

```diff
@@ -36,3 +36,17 @@ func usernsRemap() bool {
 	}
 	return false
 }
+
+// mountDevMapper checks if the Docker storage driver is Btrfs or ZFS
+func mountDevMapper() bool {
+	storage := ""
+	cmd := exec.Command("docker", "info", "-f", "{{.Driver}}")
+	lines, err := exec.CombinedOutputLines(cmd)
+	if err != nil {
```

Comment:
i'm not sure if this is the right approach here, but at least it's consistent.

this needs to get refactored anyhow, kind bootstrap has a lot of unnecessary complications I want to clobber after #148

Reply:
is consistent and is less disruptive, in case of failure it works as usual 
🤷‍♂